controllers/dynakube/dynakube_controller.go

```diff
@@ -260,6 +260,16 @@ func (r *ReconcileDynaKube) reconcileRouting(rec *utils.Reconciliation, dtc dtcl
 		if err := r.ensureDeleted(&sts); rec.Error(err) {
 			return false
 		}
+
+		ser := corev1.Service{
```

Comment:
I don't actually care, but just so you know that the short name for services in Kubernetes is `svc`.